__tests__/helpers.ts

```diff
@@ -1,5 +1,5 @@
 export const expectToHaveBeenCalledWithAuth0ClientParam = (mock, expected) => {
-  const [[url]] = (<jest.Mock>mock).mock.calls;
+  const url = (<jest.Mock>mock).mock.calls[mock.mock.calls.length - 1][0];
```

Comment:
The reason the original code was not working was because you have moved the `describe('logout()')` outside of the `describe('Auth0')`, meaning the mocks are not cleared and the test do not work when running in isolation. 

Nesting the describe blocks correctly again allows to revert this change while all tests work.

Reply:
Thanks for catching this. I'm not sure how this occurred, but certainly the cause of the problem.